Makefile

```
@@ -168,32 +171,32 @@ generate:
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 multiarch-build-push: ## Build and push a multi-arch contour-operator container image to the Docker registry
-	docker buildx build \
-		--platform $(IMAGE_PLATFORMS) \
+	"$(CONTAINER_ENGINE)" buildx build \
+		--platform "$(IMAGE_PLATFORMS)" \
 		--build-arg "BUILD_VERSION=$(BUILD_VERSION)" \
 		--build-arg "BUILD_BRANCH=$(BUILD_BRANCH)" \
 		--build-arg "BUILD_SHA=$(BUILD_SHA)" \
-		$(DOCKER_BUILD_LABELS) \
-		$(IMAGE_TAGS) \
+		"$(DOCKER_BUILD_LABELS)" \
+		"$(IMAGE_TAGS)" \
 		--push \
 		.
 
 container: ## Build the contour-operator container image
 container: test
-	docker build \
+	"$(CONTAINER_ENGINE)" build \
 		--build-arg "BUILD_VERSION=$(BUILD_VERSION)" \
 		--build-arg "BUILD_BRANCH=$(BUILD_BRANCH)" \
 		--build-arg "BUILD_SHA=$(BUILD_SHA)" \
-		$(DOCKER_BUILD_LABELS) \
+		"$(DOCKER_BUILD_LABELS)" \
 		$(shell pwd) \
-		--tag $(IMAGE):$(VERSION)
+		--tag "$(IMAGE):$(VERSION)"
 
 push: ## Push the contour-operator container image to the Docker registry
 push: container
-	docker push $(IMAGE):$(VERSION)
-ifeq ($(TAG_LATEST), true)
-	docker tag $(IMAGE):$(VERSION) $(IMAGE):latest
-	docker push $(IMAGE):latest
+	"$(CONTAINER_ENGINE)" push "$(IMAGE):$(VERSION)"
+ifeq ("$(TAG_LATEST)", true)
+	"$(CONTAINER_ENGINE)" tag "$(IMAGE):$(VERSION) $(IMAGE)":latest
```

This should be the following, right?
```suggestion
	"$(CONTAINER_ENGINE)" tag "$(IMAGE):$(VERSION)" "$(IMAGE):latest"
```